pkg/controller/iam/role/controller_test.go

```diff
@@ -20,6 +20,8 @@ import (
 	"context"
 	"testing"
 
+	"github.com/google/go-cmp/cmp/cmpopts"
+
 	"github.com/aws/aws-sdk-go-v2/aws"
```

Comment:
```suggestion
	"github.com/google/go-cmp/cmp/cmpopts"
	"github.com/aws/aws-sdk-go-v2/aws"
```

Reply:
actually, my local fmt reverts this change back... could you please give me the fmt configuration you're using?